### DISCUSSION TITLE
Visualisation results doesn't match experimental ebsd data

### DISCUSSION THREAD

**Discussion by dikshanshs** [neperfepx/neper#283] [Q&A] [answered]
**@rquey Sir,
                 When I tried visualising EBSD data using Neper following results were obtained. which doesn't match the experimental microstructure why?** (colouring mismatched and image is inverted)

![image](https://user-images.githubusercontent.com/85054451/129528492-2a78d8a8-f4d0-4832-83ef-fd6a4d8e178a.png)

**Comment by rquey** [comment, marked as answer]
EBSD software packages often assume that the origin is at the top left (EBSD scanning from top to bottom) while Neper assumes that the origin is at the bottom left. You can change the image y axis in Neper using `-camerasky` (problably `-camerasky 0:-1:0`).

For the orientations, I don't really know. I would check the colors produced by the two codes for "simple" orientations ((0,0,0), etc.) to try to find what is wrong.

**EDIT** Please keep me posted if you give it a try.

**Comment by yiyuxingchen** [comment]
@dikshanshs 
Can you tell me the steps to visualize EBSD using neper?
Thanks in advance

**Reply by rquey** [reply]
See https://neper.info/tutorials/ebsd_process.html.
